src/args.rs

```diff
@@ -98,6 +98,14 @@ pub struct Args {
         help = "Regex filter to exclude saving pages that match this expression"
     )]
     pub exclude: Regex,
+
+    /// Decides if we should bail out on download error (like, too many redirects)
+    #[structopt(
+        short,
+        long,
+        help = "If false, program will quit if error persists after number of retries. If true, it will skip problematic page"
```

Comment:
I feel like the help string is a bit too long and hard to understand

```suggestion
        help = "Flag to enable or disable exit on error"
```